File: Benchmarking/src/gen632.cpp
  #include <vector>
  #include <iostream>
  #include <chrono>
  __attribute__((noinline))
  double f0(const std::vector<double> &v0, const double &v1) {
    const double v3 = v1 * v1;
    std::vector<double> v7 = v0;
    const double v6 = v7[0];
    v7[1] = v3;
    const double v10 = v7[1];
    const double v16 = v0[0];
    v7[0] = v16;
    const double v18 = v0[0];
    const double v14 = v0[0];
    std::vector<double> v29 = v0;
    v29[0] = v6;
    const double v30 = v29[1];
    v7[1] = v14;
    v29[0] = v10;
    const double v22 = v7[1];
    std::vector<double> v165 = v29;
    const std::vector<std::vector<double>> v56 { v165, v165 };
    double v87 = v18;
    v7[1] = v87;
    v165[0] = v87;
    const std::vector<double> v82 = v56[1];
    const double v95 = v82[0];
    v7[1] = v22;
    v165[1] = v30;
    return v95;
  }
  int main() {
    std::vector<double> v0({ 0.0, 1.0 });
    double v1(2.0);
    auto start = std::chrono::high_resolution_clock::now();
    double result;
    for (int i = 0; i < 1000; i ++) {
      result = f0(v0, v1);
    }
    auto end = std::chrono::high_resolution_clock::now();
    double value = *((double*) &result);
    std::cout << value << "\n";
    std::cout << std::chrono::duration_cast<std::chrono::nanoseconds>(end-start).count();
    std::cout << "\n";
    return 0;
  }

File: Benchmarking/src/gen632.swift
  import Dispatch
  @inline(never)
  func f0(_ v0: [Double], _ v1: Double) -> Double {
    let v3: Double = v1 * v1
    var v7: [Double] = v0
    let v6: Double = v7[0]
    v7[1] = v3
    let v10: Double = v7[1]
    let v16: Double = v0[0]
    v7[0] = v16
    let v18: Double = v0[0]
    let v14: Double = v0[0]
    var v29: [Double] = v0
    v29[0] = v6
    let v30: Double = v29[1]
    v7[1] = v14
    v29[0] = v10
    let v22: Double = v7[1]
    var v165: [Double] = v29
    let v56: [[Double]] = [v165, v165]
    var v87: Double = v18
    v7[1] = v87
    v165[0] = v87
    let v82: [Double] = v56[1]
    let v95: Double = v82[0]
    v7[1] = v22
    v165[1] = v30
    return v95
  }
  func benchmark() {
    let v0: [Double] = [0.0, 1.0]
    let v1: Double = 2.0
    let start = DispatchTime.now().uptimeNanoseconds
    var result: Double = 2.0
    for _ in 1...1000 {
      result = f0(v0, v1)
    }
    let end = DispatchTime.now().uptimeNanoseconds
    print(result)
    print(end - start)
  }
  benchmark()
